backend/LexBoxApi/GraphQL/LexQueries.cs

```diff
@@ -126,6 +144,40 @@ public IQueryable<User> Users(LexBoxDbContext context)
         };
     }
 
+    public async Task<OrgMemberDto?> OrgMemberById(LexBoxDbContext context, LoggedInContext loggedInContext, Guid orgId, Guid userId)
+    {
+        var requestingUserId = loggedInContext.User.Id;
+        var requestingUser = await context.Users.Include(u => u.Organizations).Where(u => u.Id == requestingUserId).FirstOrDefaultAsync();
+        if (requestingUser is null) return null;
+
+        var isOrgAdmin = requestingUser.Organizations.Any(om => om.OrgId == orgId && om.UserId == requestingUserId && om.Role == OrgRole.Admin);
+        var allowed = isOrgAdmin || requestingUser.IsAdmin;
+        if (!allowed) return null;
```

Comment:
is there a reason we can't use `permissions.CanEditOrg(orgId)` instead and remove all the code before this line?

Reply:
I believe I added the CanEditOrg permission method later than when I wrote this code, so I didn't use it at the time and never thought about it later. Fixed in commit c9778144.